docs/technical_reference/costing/crystallizer.rst

```diff
@@ -39,7 +39,7 @@ The mass-based capital cost is dependent upon the mass of solid crystals produce
 
     .. math::
 
-        C_{cap,tot} = IEC * Cost_{ref} * (\frac{S}{size_{ref}})^{n}
+        C_{cap,tot} = IEC * Cost_{ref} * \left( \frac{S}{size_{ref}} \right)^{n}
```

Comment:
I imagine this change can be made in many places throughout the documentation, not just in the costing files. I understand addressing this would be quite tedious, so it's not necessarily something that has to be done in this PR.